taskt/Core/Automation/Commands/MergeDataTableCommand.cs

```diff
@@ -0,0 +1,156 @@
+        [Attributes.PropertyAttributes.PropertyDescription("Please indicate the Destination DataTable Name")]
+        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        [Attributes.PropertyAttributes.InputSpecification("Enter an existing DataTable to apply merging into.")]
+        [Attributes.PropertyAttributes.SampleUsage("**myDestDataTable**")]
+        [Attributes.PropertyAttributes.Remarks("")]
+        public string v_Destination_DataTableName { get; set; }
+
+        [XmlAttribute]
+        [Attributes.PropertyAttributes.PropertyDescription("Please indicate the Source DataTable Name")]
+        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        [Attributes.PropertyAttributes.InputSpecification("Enter an existing DataTable to merge into another one.")]
+        [Attributes.PropertyAttributes.SampleUsage("**mySrcDataTable**")]
+        [Attributes.PropertyAttributes.Remarks("")]
+        public string v_Source_DataTableName { get; set; }
+
+        [XmlAttribute]
+        [Attributes.PropertyAttributes.PropertyDescription("Select Missing Schema Action")]
+        [Attributes.PropertyAttributes.PropertyUISelectionOption("Add")]
+        [Attributes.PropertyAttributes.PropertyUISelectionOption("AddWithKey")]
+        [Attributes.PropertyAttributes.PropertyUISelectionOption("Error")]
+        [Attributes.PropertyAttributes.PropertyUISelectionOption("Ignore")]
+        [Attributes.PropertyAttributes.InputSpecification("Select any Missing Schema Action")]
+        [Attributes.PropertyAttributes.SampleUsage("**mySchemaAction**, **{mySchemaAction}**")]
+        [Attributes.PropertyAttributes.Remarks("")]
+        public string v_MissingSchemaAction { get; set; }
+
+        [XmlIgnore]
+        [NonSerialized]
+        public ComboBox SchemaActionComboBox;
+
+        public MergeDataTableCommand()
+        {
+            this.CommandName = "MergeDataTableCommand";
+            this.SelectionName = "Merge DataTable";
+            this.CommandEnabled = true;
+            this.CustomRendering = true;
+            this.v_MissingSchemaAction = "Add";
+        }
+
+        public override void RunCommand(object sender)
+        {
+            /* ------------Before Merge Operation, following conditions must be checked---------------
+               
+            1. None of the (Source, Destination) DataTable Variables is null            -->     (Null Check)
+            2. Data Type of both (Source, Destination) Variables must be DataTable      -->     (Data Type Check)
+            3. Source and Destination DataTable Varibales must not be the same          -->     (Same Variable Check)
+
+             */
+
+            var engine = (AutomationEngineInstance)sender;
+            var action = v_MissingSchemaAction.ConvertToUserVariable(sender);
```

Comment:
This conversion is unnecessary given than you're never actually looking up an action in the engine's variable list. It always comes from the drop down options.

Reply:
Right.
Actually, It was overlooked.

Removed it.